### PR TITLE
Upgrade Debian to Bookworm

### PR DESCRIPTION
**Pull request by Flink**

### CODE REVIEW

File: image/base/install-imagemagick [hunk 1]
```diff
@@ -8,26 +8,24 @@ IMAGE_MAGICK_HASH="d282117bc6d0e91ad1ad685d096623b96ed8e229f911c891d83277b350ef8
 # We use debian, but GitHub CI is stuck on Ubuntu Bionic, so this must be compatible with both
```
Comment (CvX) [inline image/base/install-imagemagick:8]: That is no longer true, is it? We could remove that logic and the comment?

File: image/base/install-imagemagick [hunk 2]
```diff
@@ -8,26 +8,24 @@ IMAGE_MAGICK_HASH="d282117bc6d0e91ad1ad685d096623b96ed8e229f911c891d83277b350ef8
...
 # Install build deps
 apt -y -q remove imagemagick
 apt -y -q install git make gcc pkg-config autoconf curl g++ yasm cmake \
     libde265-0 libde265-dev ${LIBJPEGTURBO} x265 libx265-dev libtool \
-    libpng16-16 libpng-dev ${LIBJPEGTURBO} ${LIBWEBP} libwebp-dev libgomp1 \
+    libpng16-16 libpng-dev ${LIBJPEGTURBO} libwebp7 libwebp-dev libgomp1 \
     libwebpmux3 libwebpdemux2 ghostscript libxml2-dev libxml2-utils librsvg2-dev \
-    libltdl7-dev libbz2-dev gsfonts libtiff-dev libfreetype6-dev libjpeg-dev
+    libltdl7-dev libbz2-dev gsfonts libtiff-dev libfreetype6-dev libjpeg-dev \
+    libvips ghostscript
```
Comment (CvX) [inline image/base/install-imagemagick:21]: ghostscript is twice on the list now

File: image/base/install-imagemagick [hunk 3]
```diff
@@ -8,26 +8,24 @@ IMAGE_MAGICK_HASH="d282117bc6d0e91ad1ad685d096623b96ed8e229f911c891d83277b350ef8
...
 
-# Ubuntu doesn't like `bullseye-backports`
+# Ubuntu doesn't like `bookworm-backports`
 if cat /etc/issue | grep -qi 'Ubuntu 22'; then
   apt -y install libheif1 libaom-dev libheif-dev
```
Comment (CvX) [inline image/base/install-imagemagick:25]: this might not be needed anymore either

File: image/base/install-imagemagick (hunk 1 above)
Comment (Flink) [reply image/base/install-imagemagick:8]: Indeed, we’re using our own images on GitHub CI, will remove :+1:

File: image/base/install-imagemagick (hunk 2 above)
Comment (Flink) [reply image/base/install-imagemagick:21]: No need for this in the end, I have a PR against core though :grin: 
https://github.com/discourse/discourse/pull/26366

File: image/base/install-imagemagick [hunk 4]
```diff
@@ -5,30 +5,19 @@ set -e
...
 
-# Ubuntu doesn't like `bullseye-backports`
-if cat /etc/issue | grep -qi 'Ubuntu 22'; then
-  apt -y install libheif1 libaom-dev libheif-dev
-else
-  # Use backports instead of compiling it
-  apt -y -q install -t bullseye-backports libheif1 libaom-dev libheif-dev
-fi
+# Use backports instead of compiling it
+apt -y -q install -t bookworm-backports libheif1 libaom-dev libheif-dev
```
[the remark this answers is not in the document]
Comment (Flink) [reply image/base/install-imagemagick:20]: This is targeting `bookworm-backports` explicitly, so not sure :thinking:

[the remark this answers is not in the document]
Comment (Flink) [reply image/base/install-imagemagick:20]: Just checked and there’s no need for targeting backports explicitly anymore.

**Review by CvX — APPROVED** [review]

**Pull request merged**